src/improvements/tasks/example/sep/002-opcm-upgrade-v200/config.toml

```diff
@@ -0,0 +1,8 @@
+l2chains = [{name = "OP Sepolia Testnet", chainId = 11155420}]
```

Comment:
@maurelian @sbvegan - Can you help field the correct values for this file for Sepolia and Mainnet?